components/expression/__examples__/base.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Example from './example';
```

Comment:
@aswinshenoy since this example will be shown on the site, unfortunately we can't import `Example` like in the other storybook variants. Please copy the contents of `__examples__/example.jsx` into this file just for this variant. Apologies for not calling this out sooner, wasn't thinking about the site imports!